valhalla/meili/config.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
+
+#include <boost/property_tree/ptree.hpp>
+
+namespace valhalla {
+namespace meili {
+
+struct Config {
+  // create config with default parameters
+  Config() = default;
+  // read parameters from boost::ptree (use default values for missing params)
+  explicit Config(const boost::property_tree::ptree& params);
+  // override default config parameters
+  void Read(const boost::property_tree::ptree& params);
+
+  struct CandidateSearch {
+    // default search radius (meters) for measurements; it's used to determine maximum search radius
+    float search_radius = 50.f;
```

Comment:
For these variables that have SI units, we should add a units suffix, like `search_radius_meters`.